### PR TITLE
Add domain meta to homepage. See #237

### PR DESCRIPTION
**Pull request by ChrisHng**

### CODE REVIEW

File: docroot/config/sync/core.extension.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ module:
   dcc_email: 0
   dcc_encryption: 0
   dcc_form_alter: 0
+  dcc_google_verification_content: 0
```
Comment (danielpopdan) [inline docroot/config/sync/core.extension.yml]: Please change the module name to dcc_google_tag

**Review by danielpopdan — CHANGES_REQUESTED** [review]

File: docroot/config/sync/core.extension.yml (hunk 1 above)
Comment (ChrisHng) [reply docroot/config/sync/core.extension.yml]: Should I rename the configuration file, aswell ?

File: docroot/modules/custom/dcc_google_tag/dcc_google_tag.routing.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+dcc_google_tag.config:
+  path: '/admin/config/google-site-tag'
```
Comment (danielpopdan) [inline docroot/modules/custom/dcc_google_tag/dcc_google_tag.routing.yml]: Please rename to google-site-verification

File: docroot/modules/custom/dcc_google_tag/dcc_google_tag.routing.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  defaults:
+    _form: '\Drupal\dcc_google_tag\Form\TagConfigForm'
```
Comment (danielpopdan) [inline docroot/modules/custom/dcc_google_tag/dcc_google_tag.routing.yml]: Please rename to GoogleVerificationTagForm.

**Review by danielpopdan — CHANGES_REQUESTED** [review]

**Pull request merged**